server/sonar-web/src/main/js/libs/sonar.js

```diff
@@ -26,12 +26,19 @@ require('script!./select2-jquery-ui-fix.js');
 require('script!./inputs.js');
 require('script!./jquery-isolated-scroll.js');
 require('script!./application.js');
+var request = require('../helpers/request');
 
 window.$j = jQuery.noConflict();
 
 jQuery(function () {
   jQuery('.open-modal').modal();
 });
 
+jQuery.ajaxSetup({
+  beforeSend: function (jqXHR) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use shorthand for method "beforeSend". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3498)